src/features/core/cookieConsent/CookieConsentFactory.js

```diff
@@ -17,13 +17,13 @@ class CookieConsentFactory {
       //     sourceConfig.cookieConsentManager &&
       // sourceConfig.cookieConsentManager.oneTrust &&
       // sourceConfig.cookieConsentManager.oneTrustenabled) {
-
-      return new OneTrust();
+      const oneTrust = new OneTrust();
+      return oneTrust.isInitialized ? oneTrust : null;
```

Comment:
The initialized validation should happen within the APIs and return appropriate data.

Reply:
Done